src/components/PageSections/ContextualNav/index.js

```diff
@@ -23,7 +23,10 @@ const ContextualNav = ({ parent, relatedTo, offeredBy, intl }) => (
               {`${intl.formatMessage(i18n.relatedTo)}: `}
             </span>
             {relatedTo.map((relatedLinkData, index) => (
-              <Link to={`/${intl.locale}${relatedLinkData.url}`} key={relatedLinkData.url}>
+              <Link
```

Comment:
made an issue for prettier stuff https://github.com/cityofaustin/techstack/issues/4460

Reply:
Maybe let's start holding off on prettier until it's part of the CI flow for every PR.

Reply:
@Niicck should I revert this back to the old way then for the time being?

Reply:
nah, that sounds like a pain. Just going forward from now.